cfgov/unprocessed/css/organisms/info-unit-group.less

```diff
@@ -58,13 +58,50 @@
     .content-l {
         margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
     }
+    
+    > .content-l:first-child {
+        margin-top: 0;
+    }
 
     .content-l_col-1 + .content-l_col-1 {
         // Adds margin top to info units that follow another
         margin-top: unit( @grid_gutter-width / @base-font-size-px, em ) !important;
     }
+    
+    .content-l__top-divider {
+        .content-l_col-1 + .content-l_col-1 {
+            // Adds top divider between single-column info units
+            .grid_column__top-divider;
+        }
+    }
+    
+    .content-l__top-divider + .content-l__top-divider {
+        // Adds top divider between multi-column info unit rows
+        .grid_column__top-divider;
+        margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
+    }
+    
+    .respond-to-max( @bp-xs-max, {
+        // Adds top divider between info unit cols on small screens
+        .content-l__top-divider {
+            .content-l_col + .content-l_col {
+                .grid_column__top-divider;
+                margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
+            }
+        }
+    });
```

Comment:
Minor, add space within paren here.

Reply:
Should be fixed in https://github.com/cfpb/cfgov-refresh/pull/3476/commits/4c649330961585504690e2ccfe018ae41b4e57cc